regression-test/suites/mtmv_p0/same_column_name_check/same_column_name_check.groovy

```diff
@@ -0,0 +1,102 @@
+package same_column_name_check
```

Comment:
what's this?

Reply:
seems package name, auto genereated when create groovy package